presto-orc/src/main/java/com/facebook/presto/orc/OrcOutputBuffer.java

```diff
@@ -49,6 +49,7 @@
         extends SliceOutput
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(OrcOutputBuffer.class).instanceSize();
+    private static final int PAGE_HEADER_SIZE = 3;
```

Comment:
Is this number `3` coming from the 3-bit header from ORC spec? If yes, could you add a comment?

Reply:
Yes, these are the 3 bytes that are used for the header.

https://github.com/apache/orc/blob/028261ad6e26d8ae9f9483c8aeb76a85c7f1168b/c%2B%2B/src/Compression.cc#L186